docs/languages/java.md

```diff
@@ -151,7 +151,16 @@ One of the key advantages of VS Code is speed. When you open your Java source fi
   <source src="/docs/languages/java/intellisense.mp4" type="video/mp4">
 </video>
 
-For more details, see [Java Code Navigation and Editing](/docs/java/java-editing.md). VS Code also supports a range of [Refactoring](/docs/java/java-refactoring.md) and [Linting](/docs/java/java-linting.md) features.
+## Code snippets
+
+Visual Studio Code supports a wide range of popular Java code snippets to make you more productive, such as class/interface, syserr, sysout, if/else, try/catch, static main method. Using information from Java language server, it also provides a preview of the code snippet during the selection.
+
+For example, typing "**sout**" or "**sysout**" will directly produce a code snippet for `System.out.println()`; 
+Similarly, typing "**main**" or "**psvm**" will generate a code snippet for `public static void main(String[] args) {}`;
+
+We support a wide range of code snippet shortcuts and postfix completion features. To see the complete list, see [Code Snippets](/docs/java/java-editing.md#code-snippets). VS Code also supports a range of [Refactoring](/docs/java/java-refactoring.md) and [Linting](/docs/java/java-linting.md) features.
+
+![Code Snippet](images/java-editing/code-snippet.png)
```

Comment:
this image already exists there

Reply:
It does but you won't be able to reach it via the `images/java-editing/code-snippet.png` path.
You can see that the image doesn't resolve if you use GitHub "View file".
![image](https://user-images.githubusercontent.com/12818376/206738227-fe809799-a341-4510-805e-778e8a171555.png)

It's also good practice to not reach across doc areas for images, as it makes makes moving a topic easier since the images are local to the topic. 
I'll make a copy of the image in a follow up PR.

Reply:
https://github.com/microsoft/vscode-docs/pull/5846